src/info/utils/mod.rs

```diff
@@ -24,20 +24,26 @@ where
 }
 
 fn to_human_time(time: Time) -> String {
-    let since_epoch_duration = SystemTime::now()
-        .duration_since(SystemTime::UNIX_EPOCH)
-        .unwrap();
+    let duration = match diff_gix_time(SystemTime::now(), time) {
+        Ok(d) => d,
+        Err(s) => return s,
+    };
+    let ht = HumanTime::from(-(duration.as_secs() as i64));
+    ht.to_string()
+}
 
-    let ts = Duration::from_secs(match time.seconds.try_into() {
+/// Gets the duration between `now` and `time`. Returns `Err` if this cannot be calculated.
+fn diff_gix_time(now: SystemTime, time: Time) -> Result<Duration, String> {
+    let since_epoch_duration = now.duration_since(SystemTime::UNIX_EPOCH).unwrap();
+    let ts = Duration::from_secs(match (time.seconds + i64::from(time.offset)).try_into() {
```

Comment:
I'm a bit perplexed. Why do we need to consider the offset here? This method should simply return the difference between two UTC timestamps (without timezone): `now (since epoch in UTC+0) - commit.time (since epoch in UTC+0`.

Sorry if I'm mistaken.

Reply:
I made this with the assumption that #1306 was caused by differing timezones, which was an incorrect assumption, which made me assume that we were comparing a timezone-aware time to a naive time. Right now, personally, I am not sure if `commit.time.seconds` is UTC+0 or if it's a naive time that needs `offset` to get the real time.

Reply:
Sorry, I also didn't catch it. `commit.time.seconds` is UTC. The [doc-string there](https://docs.rs/gix-date/latest/gix_date/struct.Time.html#structfield.seconds) unfortunately only implies it ("time in seconds since epoch."). I have fixed this [here](https://github.com/Byron/gitoxide/blob/554ec1c448cbf8da07d3a24b22696ab1c7c1dcb0/gix-date/src/lib.rs#L26).

Reply:
Thanks! I guess this PR should be closed, then.